resources/windows/wix/service.component.wxs.erb

```diff
@@ -0,0 +1,55 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+  <%- get_services.each do |service| -%>
+    <%- service.options.each do |component| -%>
```

Comment:
I'm thinking this is accounting for the manual startup, the disabled startup and the automatic startup for EACH service.